lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1776,6 +1798,14 @@ public function get_collection_params() {
 				'default'           => array(),
 			);
 		}
+
+		if ( 'post' === $this->post_type ) {
+			$params['sticky'] = array(
+				'description' => __( 'Limit result set to items that are sticky.' ),
+				'type'        => 'boolean',
```

Comment:
Add in the `rest_validate_request_arg` and `rest_sanitize_request_arg` for validate and sanitize callbacks and this patch will be da hotness!